### PR TITLE
Star_aln.t

### PR DESCRIPTION
**Pull request by J35P312**
Good morning!
I have made a test for star_aln, the test seems to be bug free! =P
I also fixed a bug in the Star.pm file

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
Great! @northwestwitch can you do a first review?

**Comment by northwestwitch** [comment]
Ok! I check and get back to you!

File: lib/MIP/Program/Alignment/Star.pm [hunk 1]
```diff
@@ -26,7 +26,7 @@ BEGIN {
     our $VERSION = 1.00;
 
     # Functions and variables which can be optionally exported
-    our @EXPORT_OK = qw{ star_aln, star_genome_generate };
+    our @EXPORT_OK = qw{ star_aln star_genome_generate };
```
Comment (northwestwitch) [inline lib/MIP/Program/Alignment/Star.pm]: Here is fine, but fix the indentation 
: $outfile_name_prefix       => Prefix of the output files (remember to end with a ".")

File: t/star_aln.t [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+diag(   q{Test star_aln from Star.pm v}
+      . $MIP::Program::Alignment::Star::VERSION
+      . $COMMA
+      . $SPACE . q{Perl}
+      . $SPACE
+      . $PERL_VERSION
+      . $SPACE
+      . $EXECUTABLE_NAME );
+
+## Constants
```
Comment (northwestwitch) [inline t/star_aln.t]: Move them in alphabetical order

File: t/star_aln.t [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+        input           => catfile(qw{ dir test }),
+        expected_output => q{--outFileNamePrefix}
+          . $SPACE
+          . catfile(qw{ dir test }),
+    },
+    out_sam_strand_field => {
+        input           => q{intronMotif},
+        expected_output => q{--outSAMstrandField} . $SPACE . q{intronMotif},
+    },
+    out_sam_type => {
```
Comment (northwestwitch) [inline t/star_aln.t]: It's already a required argument, do you need it in the specific ones as well?

Comment (henrikstranneheim) [reply t/star_aln.t]: yes, to test this parameter specifically

**Comment by henrikstranneheim** [comment]
👍

**Comment by J35P312** [comment]
Hello there!
Thanks for the review northwestwitch ! =P.
I have fixed he indentation of the star_aln header and the order of the constants

**Pull request merged**